libraries/ESP32/examples/Camera/CameraWebServer/CameraWebServer.ino

```diff
@@ -11,6 +11,7 @@
 #define CAMERA_MODEL_WROVER_KIT // Has PSRAM
 //#define CAMERA_MODEL_ESP_EYE // Has PSRAM
 //#define CAMERA_MODEL_M5STACK_PSRAM // Has PSRAM
```

Comment:
feel free to add better explanation to both boards, so it is easier for users to distinguish. Like "manufactured before ..." or something else.

Reply:
i think it is enough. i add the "version B" note. our user will know it.  actually , we just saled few old version. most is version B.  that is why i hope change it not add a new version.   but add a version B define is a good solution. could you meger the PR.?